tests/test_scylla.c

```diff
@@ -0,0 +1,143 @@
+/*
+ * Copyright (C) 2019 BiiLabs Co., Ltd. and Contributors
+ * All Rights Reserved.
+ * This is free software; you can redistribute it and/or modify it under the
+ * terms of the MIT license. A copy of the license can be found in the file
+ * "LICENSE" at the root of this distribution.
+ */
+
+#include "storage/scylla_api.h"
+#include "test_define.h"
+
+char* host = "node0.puyuma.org";
```

Comment:
Don't hardcode `node0` here. Instead, specify via configuration file and/or command options.